index.html

```diff
@@ -23,9 +23,13 @@
 </head>
 <body>
    <section class="body">
-    <section class="sect">
 
-      
+    <section id="popup_mobile" class="multi1"></section>
+
```

Comment:
[ OPTIONAL ] Icons inside pop-up buttons are not aligned vertically with text. Kindly see the screenshot below and fix the alignment of the icons.
![icons](https://user-images.githubusercontent.com/35504149/132014563-f57e0b40-df3a-4526-9fe4-ccc694415f51.png)

Reply:
I'll get right on that.